docs/source/index.rst

```diff
@@ -10,7 +10,26 @@
 JupyterLab Documentation
 ########################
 
-JupyterLab is the next-generation web-based user interface for Project Jupyter. `Try it on Binder <https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/3818244?urlpath=lab/tree/demo>`__.  JupyterLab follows the Jupyter `Community Guides <https://jupyter.readthedocs.io/en/latest/community/content-community.html>`__.
+Welcome to the **JupyterLab** documentation site. **JupyterLab** is a highly
+extensible, feature-rich notebook authoring application and editing environment,
+and is a part of `Project Jupyter <https://docs.jupyter.org/en/latest/>`_, a
+large umbrella project centered around the goal of providing tools for interactive
+computing with `computational notebooks <https://docs.jupyter.org/en/latest/#what-is-a-notebook>`_.
+
+A `computational notebook <https://docs.jupyter.org/en/latest/#what-is-a-notebook>`_
+is a shareable document that combines computer code, plain language descriptions,
+data, rich visualizations like 3D models, charts, graphs and figures, and interactive
+controls. A notebook, along with an editor like **JupyterLab**, provides a fast interactive
+environment for prototyping and explaining code, exploring and visualizing data, and sharing
+ideas with others.
+
+**JupyterLab** is a sibling to other notebook authoring applications under
```

Comment:
If we mention "other notebook authoring applications under the Project Jupyter umbrella" in plural, should we name them and provide links to them? Otherwise it's not clear we are we talking about here

Reply:
After thinking about it, other than JupyterLab and Notebook I can't name "other notebook authoring applications under the Project Jupyter umbrella". For example, Voila and Nbcovert are more of converters and Jupyter Book is a book building application.

Reply:
As discussed here https://github.com/jupyter/notebook/pull/6877#discussion_r1204298943, it would be nice to mention 1 or 2 extras of Notebook authoring applications.

Reply:
As discussed here https://github.com/jupyter/notebook/pull/6877#discussion_r1204298943, it would be nice to mention 1 or 2 extras of Notebook authoring applications like Jupyter Desktop or JupyterLite.